mesonbuild/dependencies/misc.py

```diff
@@ -671,3 +796,159 @@ def get_methods(self):
             return [DependencyMethods.PKGCONFIG, DependencyMethods.CUPSCONFIG, DependencyMethods.EXTRAFRAMEWORK]
         else:
             return [DependencyMethods.PKGCONFIG, DependencyMethods.CUPSCONFIG]
+
+
+# Generated with boost_names.py
+# (directory name, human readable name, library names)
+BOOST_MODULES = [('accumulators', 'Accumulators', []),
+ ('algorithm', 'Algorithm', []),
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->